automatic/tv-browser/update.ps1

```diff
@@ -12,26 +12,19 @@ function global:au_SearchReplace {
       "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType32)"
       "(?i)(^\s*checksum(32)?\:).*"       = "`${1} $($Latest.Checksum32)"
     }
-    ".\tools\chocolateyInstall.ps1" = @{
-      "(?i)(^\s*file\s*=\s*`"[$]toolsPath\\).*"   = "`${1}$($Latest.FileName32)`""
```

Comment:
Why do you keep removing this every single time you touch a package?

Do you enjoy creating problems further down the line? Or unneeded questions by users of the package?

Reply:
Lets agree to disagree on that one, then.